stack-scholar/recipes/deploy.rb

```diff
@@ -1,6 +1,23 @@
 include_recipe 'php-fpm::service'
 include_recipe 'nginx-app::service'
 
+#
+# An attempt to make sure that the php prefix given is the prefix
+# chosen for php-cli at the end of an install
+php_package_prefix = node['php']['ppa']['package_prefix'].gsub(/php/, '')
```

Comment:
@till yea, thats what I am working against, the problem is that it's being executed and something later is installing php-cli 7 so I need to figure out to refer to this in "later timing"